style.css

```diff
@@ -111,3 +112,269 @@ body {
   transition: 1s ease-in-out;
   cursor: pointer;
 }
+
+.works-section {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 114px 24px;
+}
+
+.cards {
+  display: flex;
+  flex-direction: column;
+  gap: 88px;
+}
+
+.card {
+  width: 311px;
+  height: 552px;
+  border: 1px solid #dfe1e6;
+  border-radius: 16px;
+  display: grid;
+  flex-direction: column;
+  align-items: center;
+  padding: 16px;
+  gap: 12px;
+}
```

Comment:
- [ ] Good work positioning the elements inside your cards using display `flex`. ✔️ However, please kindly ensure to layout the elements inside your cards using display `flex` instead of display `grid` as requested in the requirement below. 👇🏽
>In order to lay out the elements in "Works" section you should use CSS Grid and Flexbox combined (it is mandatory that you use both). Use CSS Grid in the main container that contains all of the cards and use Flexbox to lay out an individual card. 🤓.